pokemongo_bot/Logger.py

```diff
@@ -0,0 +1,23 @@
+try:
+    import lcd
+    lcd = lcd.lcd()
+    # Change this to your i2c address
+    lcd.set_addr(0x23)
+except:
+    lcd = False
+
+def log(string, color = 'white'):
+    colorHex = ''
+    if(color == 'green'):
+        colorHex = '92m'
+    if(color == 'yellow'):
+        colorHex = '93m'
+    if(colorHex == 'red'):
+        colorHex = '91m'
+    if colorHex == '':
```

Comment:
And why not use the logging module? I want to use it properly because it's a default choice for logging in Python. We can create a wrapper around the logging module to log to LCD too, that's no problem.